src/mono/wasm/test-main.js

```diff
@@ -509,6 +540,26 @@ const App = {
             console.error("exception thrown in", fqn);
             throw exc;
         }
+    },
+
+    create_function(...args) {
+        const code = args.pop();
+        const arg_count = args.length;
+        args.push("MONO");
```

Comment:
it's `MONO, INTERNAL, BINDING, IMPORTS, EXPORTS, Module, runArgs` above but `MONO, BINDING, INTERNAL, IMPORTS, EXPORTS` here. Is there a reason for that?

Reply:
`runArgs` is temporary thing of a startup.